### ISSUE TITLE
Dropout implementation

### ISSUE THREAD

**Issue by soaperless** [yonatandn/Uncertainty-Quantification-in-Model-Based-DL#12]
I have my own thoughts on this, but I still wanted to ask sincerely: why not use nn.Dropout to implement Dropout

**Comment by yonatandn** [comment]
As mentioned in the paper, we have followed Y. Gal implementation presented in its paper Concrete Dropout.
I suggest you read it too for more information about the dropout layer.
